src/StackExchange.Redis/KeyspaceIsolation/WrapperBase.cs

```diff
@@ -804,6 +807,25 @@ internal WrapperBase(TInner inner, byte[] keyPrefix)
             }
         }
 
+        protected ReadOnlyMemory<KeyValuePair<RedisKey, RedisValue>> ToInner(ReadOnlyMemory<KeyValuePair<RedisKey, RedisValue>> outer)
+        {
+            if (outer.Length == 0)
+            {
+                return outer;
+            }
+            else
+            {
+                KeyValuePair<RedisKey, RedisValue>[] inner = new KeyValuePair<RedisKey, RedisValue>[outer.Length];
+
+                for (int i = 0; i < outer.Length; ++i)
+                {
+                    inner[i] = ToInner(outer.Span[i]);
```

Comment:
Accessing .Span is relatively expensive; this should be done outside the loop and resused.

Reply:
The ++i is very unusual for C#, note

Reply:
I'll make the changes to cache all call sites for .Span.

The ++i was copied from line 801 of the diff. I assumed that was intentional and copied it here. Is that an incorrect usage?